### PR TITLE
added landing slide

### PR DESCRIPTION
**Pull request by gabrielperales**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -49,23 +67,19 @@ body {
 
 .sidenav {
    height: 100%; /* 100% Full-height */
-   width: 0; /* 0 width - change this with JavaScript */
+   width: 200px;
```
Comment (mcelearr) [inline css/style.css:70]: @gabrielperales Why is it better to keep the width the same and have it come in from the left?

File: js/app.js [hunk 2]
```diff
@@ -1,35 +1,17 @@
 /* Set the width of the side navigation to 250px and the left margin of the page content to 250px and add a black background color to body */
 function openNav() {
   $u('#mySidenav')
-    .removeClass('closedsidenav')
     .addClass('opensidenav');
```
Comment (mcelearr) [inline js/app.js:4]: .addClass() add the classname to the classlist for the target node.

``` javascript
  var addClass = function(className) {
    target.classList.add(className);
    return this;
  };
```

But now we're not removing 'closedsidenav' class from the classlist for the target node. So the target node still has that class?? If it has two classes, both clsoedsidenav and opensidenav, how does the CSS know which class to render?

File: css/style.css (hunk 1 above)
Comment (gabrielperales) [inline css/style.css:70]: Then thing was that while the sidenav was growing, as you can see in the following picture menu items had a weird behaviour:

captura de pantalla 2016-06-12 10 56 54

Some items had 2 lines while growing, and also part of the text was sliced/cutted...

On the other hand, shrink the main div did that our profiles were collapsed, so now, sidenav goes over the main div, it also _auto-hides_ when you click on one of the sidenav items.

File: js/app.js (hunk 2 above)
Comment (gabrielperales) [inline js/app.js:4]: Now we only have `opensidenav` class, the other one wasn't needed.
The default behaviour of our sidenav is to be hidden (or closed), that is at -200px from the left. When we add the `opensidenav` class we translate that sidenav 200px to the right, so it show up, but if we remove that class it is going to be hidden again. We only have to _toggle_ that class :-)

P.S: remember to keep it simple!

**Pull request merged**